### DISCUSSION TITLE
Issue Ingesting

### DISCUSSION THREAD

**Discussion by cla11826** [mayooear/ai-pdf-chatbot-langchain#12] [Q&A]
Hi Mayo, I am currently stuck on the ingest part. Here is the error I see when I run npm run ingest in powershell. Also, I was able to unzip, but the file is not named Notion_DB, not sure if that will cause any issues.
![image](https://user-images.githubusercontent.com/35822898/226489081-315a0d2d-80f6-4336-8505-d0663a0a7b32.png)

Here is the error when running in Bash.
![image](https://user-images.githubusercontent.com/35822898/226489161-5aeb3a62-2326-452f-a6ac-ffb294ca9654.png)

By the way, I'm pretty new to coding in general, so that may be the cause of the errors. Appreciate any help you could provide. Thanks!

**Comment by mayooear** [comment]
Hi, this is caused because Node.js isn't installed on your computer or not properly configured under your system Path.

Visit https://nodejs.org to download and install Node.js. Then after run `node -v` and `npm -v` in your terminal to verify it works.

If that doesn't work there's an issues with your system's PATH, so you can try the instructions [here](https://stackoverflow.com/questions/27864040/fixing-npm-path-in-windows-8-and-10).

**Reply by cla11826** [reply]
Hi, Thanks for the response. I was able to add Node.js, but have run into another issue. Is it looking for the file in the wrong place?

<img width="922" alt="image" src="https://user-images.githubusercontent.com/35822898/227799233-02288a98-5482-4575-b59c-3752db0c5926.png">

Appreciate any help you could provide on this as well. Thanks again!

**Comment by gasper94** [comment]
I'm getting [Error: Failed to ingest your data]. My issue does not sems to be with node but with too many requests been sent.  Is anyone able to ingest data successfully? I have checked everything: .env folder, pinecone config, I even tried doing some timeouts for each request in the ingest-data.ts file.

 response: {
    ok: false,
    status: 429,
    statusText: 'Too Many Requests',
    headers: HeadersList {
      cookies: null,
      [Symbol(headers map)]: [Map],
      [Symbol(headers map sorted)]: null
    },
    config: {
      transitional: [Object],
      adapter: [AsyncFunction: fetchAdapter],
      transformRequest: [Array],
      transformResponse: [Array],
      timeout: 0,
      xsrfCookieName: 'XSRF-TOKEN',
      xsrfHeaderName: 'X-XSRF-TOKEN',
      maxContentLength: -1,
      maxBodyLength: -1,
      validateStatus: [Function: validateStatus],
      headers: [Object],
      method: 'post',
      data: ....
    url: 'https://api.openai.com/v1/embeddings'
    },
    request: Request {
      [Symbol(realm)]: [Object],
      [Symbol(state)]: [Object],
      [Symbol(signal)]: [AbortSignal],
      [Symbol(headers)]: [HeadersList]
    },
    data: { error: [Object] }
  },
  isAxiosError: true,
  toJSON: [Function: toJSON]
}
/mnt/c/Users/Ulises/Desktop/chat/gpt4-pdf-chatbot-langchain/scripts/ingest-data.ts:53
    throw new Error('Failed to ingest your data');
          ^


[Error: Failed to ingest your data]

Node.js `v18.15.0`

**Reply by gasper94** [reply]
For those having issues with this. I found out I had reached my openai credit limit. I upgraded to pay as you go and everything is working properly now.
